--- gksu-2.0.2/gksu/gksu.c.orig	2009-03-04 19:25:32.000000000 +0100
+++ gksu-2.0.2/gksu/gksu.c	2023-03-25 15:59:28.118988248 +0100
@@ -148,7 +148,7 @@ gk_dialog (GtkMessageType type, gchar *f
 
   diag_win = gtk_message_dialog_new_with_markup (NULL, GTK_DIALOG_MODAL,
 						 type, GTK_BUTTONS_CLOSE,
-						 msg);
+						 "%s", msg);
 
   gtk_signal_connect_object (GTK_OBJECT(diag_win), "delete_event",
 			     GTK_SIGNAL_FUNC(gtk_main_quit),
@@ -176,12 +176,8 @@ set_description_from_desktop (GksuContex
   g_key_file_load_from_file (desktop, file_name, G_KEY_FILE_NONE, &error);
   if (error)
     {
-      gchar *error_msg;
-
-      error_msg = g_strdup_printf ("Could not load desktop file: %s",
+      g_warning ("Could not load desktop file: %s",
 				   error->message);
-      g_warning (error_msg);
-      g_free (error_msg);
       g_error_free (error);
       g_key_file_free (desktop);
       return;
