diff -urN aa/gksu-1.0.3/docs/Makefile.am gksu-1.0.3/docs/Makefile.am
--- aa/gksu-1.0.3/docs/Makefile.am	2004-01-14 06:54:17.000000000 +0100
+++ gksu-1.0.3/docs/Makefile.am	2004-02-15 23:16:36.162221760 +0100
@@ -31,8 +31,8 @@
 FIXXREF_OPTIONS=
 
 # Used for dependencies.
-HFILE_GLOB=$(top_srcdir)/libgksu/*.h $(top_srcdir)/common/*.h
-CFILE_GLOB=$(top_srcdir)/libgksu/*.c $(top_srcdir)/common/*.c
+HFILE_GLOB=$(top_srcdir)/libgksu/*.h $(top_srcdir)/gksu/*.h
+CFILE_GLOB=$(top_srcdir)/libgksu/*.c $(top_srcdir)/gksu/*.c
 
 # Header files to ignore when scanning.
 IGNORE_HFILES= defines.h pixmaps.h config.h
